### PR TITLE
Remove unused use statements

### PR DESCRIPTION
**Pull request by bcrowe**

### CODE REVIEW

File: src/Network/CorsBuilder.php [hunk 1]
```diff
@@ -14,7 +14,6 @@
  */
 namespace Cake\Network;
 
-use Cake\Network\Response;
```
Comment (antograssiot) [inline src/Network/CorsBuilder.php]: @bcrowe extra blank line that causes cs issue here

Comment (bcrowe) [inline src/Network/CorsBuilder.php]: Thanks, removed!

File: src/Error/BaseErrorHandler.php [hunk 2]
```diff
@@ -15,7 +15,6 @@
 namespace Cake\Error;
 
 use Cake\Core\Configure;
-use Cake\Error\PHP7ErrorException;
```
Comment (ADmad) [inline src/Error/BaseErrorHandler.php]: PHP7ErrorException is used on line 161.

File: src/Error/ExceptionRenderer.php [hunk 3]
```diff
@@ -19,7 +19,6 @@
 use Cake\Core\Configure;
 use Cake\Core\Exception\Exception as CakeException;
 use Cake\Core\Exception\MissingPluginException;
-use Cake\Error\PHP7ErrorException;
```
Comment (ADmad) [inline src/Error/ExceptionRenderer.php]: PHP7ErrorException is used on line 103.

File: src/Error/BaseErrorHandler.php (hunk 2 above)
Comment (dereuromark) [inline src/Error/BaseErrorHandler.php]: Yeah I wonder what IDE told you that they are unused :)

Comment (bcrowe) [inline src/Error/BaseErrorHandler.php]: php-cs-fixer :sob:

Comment (dereuromark) [inline src/Error/BaseErrorHandler.php]: Strange, I also used that in the past and it worked fine.

**Pull request merged**